charts/rancher-logging/0.0.1/charts/fluentd/templates/role.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.podSecurityPolicy.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
It relates to `pod-security-policy.yaml`, merging into `pod-security-policy.yaml`.